test/lib/ufe/testParentCmd.py

```diff
@@ -258,8 +257,10 @@ def testParentAbsolute(self):
         # Cube world y coordinate is currently 0.
         self.assertAlmostEqual(cubeWorld.matrix[3][1], 0)
 
-        # Move the parent
-        ufe.GlobalSelection.get().append(cylinderItem)
+        # Move only the parent.
```

Comment:
Test was failing when run with sabrih/MAYA-112126/parent_selection because this new change properly sets the selection onto the parented child item, rather than on nothing (current behavior).  Using append instead of replace meant that both the parent and the child were moved, which is not what the test intended to do.  This change is safe with the current Maya master, and will allow sabrih/MAYA-112126/parent_selection to be merged in Maya.